ci-operator/jobs/openshift/library/openshift-library-master-periodics.yaml

```diff
@@ -11,10 +11,10 @@ periodics:
   name: periodic-openshift-library-import
   reporter_config:
     slack:
-      channel: '#forum-build-api'
+      channel: '#forum-app-services'
```

Comment:
Oh OK, send that to forum-appsvc-notify

Reply:
Updated. @dperaza4dustbit I believe you have the ability to lgtm.